content/blog/2018-03-23-hydroscoper.md

```diff
@@ -0,0 +1,157 @@
+---
+slug: "hydroscoper"
+title: ὕδωρ + σκοπῶ
+package_version: 1.0.0
+authors:
+  - name: Konstantinos Vantas
+    url: https://kvantas.github.io/
+date: 2018-03-23
+categories: blog
+topicid: 666666666
+tags:
+  - r
+  - community
+  - software
+  - review
+  - package
+  - data-access
+  - hydrology
+  - Greece
+  - hydroscope
+output:
+  md_document:
+    variant: markdown_github
+    preserve_yaml: true
+---
+
+> Hydrology is a concept to unify statistics, data analysis and numerical models in order to understand and analyze the endless circulation of water between the earth and its atmosphere.
+
+That's a lot alike Data Science, isn't it? Hydrologic Processes evolve in space and time, are extreme complex and we may never comprehend them. For this reason Hydrologists use models where their inputs and outputs are measurable variables: climatic and hydrologic data, land uses, vegetation coverage, soil type etc.  
+
+Not so many years ago, I used to struggle for these data: travelling 500 km away searching in underground storages, requesting from peers and institutes or even use sophistries to get them. Nowadays, hydrologic data come from a variety of sources such as satellites, online weather stations, national climatic databases, etc. But problems never end: these data have a variety of types and formats, errors, noise and a big amount of missing values. 
+
+In Greece, the Hydroscope (ὕδωρ + σκοπῶ) project started in the 1990s, aimed to organize data belonging to various institutions and to create a database of hydrometeorological information. Today this database, [Hydroscope](http://www.hydroscope.gr/), is available online. Unfortunately, all the data in Hydroscope are in Greek and there are no convenient look-up tables. 
+
+## The hydroscoper package
+
+`hydroscoper` is an R package designed to provide functionality for automatic retrieval and translation of Hydroscope's data to English.  Furthermore, the internal data sets can be used to run queries on the available weather stations, gauging stations and time series, reducing the time needed for downloading and data wrangling. 
+
+### Installation
+
+The package is available in [CRAN](https://cran.r-project.org/web/packages/hydroscoper/index.html) and can be installed with:
+
+```r
+install.packages("hydroscoper")
+```
+
+The development version from GitHub can be installed using [devtools](https://cran.r-project.org/web/packages/devtools/index.html):
+```r
+devtools::install_github("ropensci/hydroscoper")
+```
+
+### Using the package
+
+The following code is a demonstration of how to download and plot the available data from a specific station. First of all, the package must be loaded:
+```r
+load(hydroscoper)
+```
+The time series that will be used come from the station `AG. BASILEIOS`, in the Greek water division `GR13`. The station `AG. BASILEIOS` appears to have two different IDs and two different owners (maybe is an inconsistency in Hydroscope's databases):
```

Comment:
"appears has two different IDs" -> appears to have

Reply:
Is this having two IDs or two owners, and noting possible database inconsistency a red herring?
That is,  by saying this it draws the reader's attention and I skimmed down to see if that would be resolved. Decide how to say it, depending on how much attention you want reader to pay